line.go

```diff
@@ -95,11 +95,21 @@ func (l *lineEdit) ClearRight() {
 }
 
 func (l *lineEdit) Bytes() []byte {
+	// Trim whitespace off the end
+	for i := l.size - 1; i >= l.pos; i-- {
+		if l.buf[i].char == ' ' {
+			l.size = i
+		} else {
+			break
+		}
+	}
+
```

Comment:
This might behave weirdly in color preservation mode. Maybe only strip if there's no vt100 on the cell? It's also simpler to `bytes.TrimRight` after generation than loop manually.